workspaces/azure-devops/packages/backend/src/index.ts

```diff
@@ -48,8 +48,8 @@ backend.add(
 
 // search plugin
 backend.add(import('@backstage/plugin-search-backend'));
-backend.add(import('@backstage/plugin-search-backend-module-catalog'));
-backend.add(import('@backstage/plugin-search-backend-module-techdocs'));
+// backend.add(import('@backstage/plugin-search-backend-module-catalog'));
+// backend.add(import('@backstage/plugin-search-backend-module-techdocs'));
```

Comment:
Shoot, these can be turned back on, or honestly removed. I had done this to speed up my testing.